src/layout/index.tsx

```diff
@@ -20,8 +20,28 @@ const Layout: React.FC<LayoutProps> = ({
 }) => {
   const footerContainer = useRef<HTMLElement>(null);
 
+  const [clickActive, setClickActive] = useState(false);
+  const burgerRef = useRef<HTMLDivElement>(null);
+
+  const handleClick = (e: Event) => {
```

Comment:
This could have a more specific name. Such as `onClickOutside` or something.

Reply:
Based on the comment bellow i fixed it with just an onClick and removed burgerRef and useEffect.

Reply:
also changed the name to onClickOutsideUl